NOTICE

```diff
@@ -17,9 +17,42 @@ under the auspices of the Eigenbase Foundation
 and released as the LucidDB project.
 
 
+================= Apache Flink =================
+Apache Flink
+Copyright 2014-2019 The Apache Software Foundation
 
 
-================= Metamarkets extendedset 1.3.10 =================
+================= Apache Hadoop =================
+Apache Hadoop
+Copyright 2009-2017 The Apache Software Foundation
+
+
+================= Apache Hive =================
+Apache Hive
+Copyright 2008-2018 The Apache Software Foundation
+
+
+================= Apache Kafka =================
+Apache Kafka
+Copyright 2019 The Apache Software Foundation
+
+
+================= Apache Lucene =================
```

Comment:
https://github.com/apache/incubator-druid/pull/3148 was raised on June 14, 2016 and the source code was adapted from the master branch. Since Lucene 6.1.0 and 6.2.0 were release on June 16 and August 24, respectively, I think the version would be 6.2.0.